soyjs/exec_test.go

```diff
@@ -1045,6 +1047,60 @@ soy.$$escapeHtml = function(arg) { return arg; };
 	}
 }
 
+func TestES6(t *testing.T) {
```

Comment:
Thanks for adding this! Are there any other edge cases you can think of?

Reply:
Not at the moment, no. If more functionality is added, we will probably have to add more tests (eg. for let, const, etc.)